language/move-prover/tests/testsuite.rs

```diff
@@ -227,11 +227,12 @@ fn cvc4_deny_listed(path: &Path) -> bool {
 fn get_flags(temp_dir: &Path, path: &Path) -> anyhow::Result<(Vec<String>, Option<PathBuf>)> {
     // Determine the way how to configure tests based on directory of the path.
     let path_str = path.to_string_lossy();
-    let (base_flags, baseline_path, modifier) = if path_str.contains("diem-framework/") {
-        (STDLIB_FLAGS, None, "std_")
-    } else {
-        (STDLIB_FLAGS, Some(path.with_extension("exp")), "prover_")
-    };
+    let (base_flags, baseline_path, modifier) =
+        if path_str.contains("diem-framework/") || path_str.contains("move-stdlib") {
```

Comment:
```suggestion
        if path_str.contains("diem-framework/") || path_str.contains("move-stdlib/") {
```

Reply:
Done, however, I do not use suggestions.